scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
> In this case, it should be OK to use require('hexo-util')

Yes, it is. But, if `hexo-util` is standart hexo module in `hexo` subdir, why need to separate this to external module?

Anyway, u tested it? All should be worked fine on clean install.
If user will install `hexo-util` as external moduke, no matter, tag will use internal `hexo-util`.

What the question, VI? I don't understand.